airflow/config_templates/config.yml

```diff
@@ -326,7 +337,7 @@ core:
       version_added: 2.3.0
       type: integer
       example: ~
-      default: ""
+      default: None
```

Comment:
Why did you change this?
It causes tests to fail
`airflow.exceptions.AirflowConfigException: Failed to convert value to int. Please check "default_task_execution_timeout" key in "core" section. Current value: "None".`

Reply:
I am checking it, somehow the config is not coming through in arguments